packages/edit-site/src/components/global-styles/font-families.js

```diff
@@ -37,7 +37,7 @@ function FontFamilies() {
 				/>
 			) }
 
-			<VStack spacing={ 3 }>
+			<VStack spacing={ 2 }>
```

Comment:
This is wrapped in another VStack inside screen-typography. Maybe we should just remove that one and add that space to this one?

Reply:
Good catch; https://github.com/WordPress/gutenberg/pull/60031/commits/a668bb745982c3a6a96e5b58fc96b89a216a2b9a